src/main/scala/bi/deep/SparkSchema.scala

```diff
@@ -0,0 +1,22 @@
+package bi.deep
+
+import org.apache.spark.sql.types.{StructField, StructType}
+
+class SparkSchema(
+                   var dimensions: Array[StructField],
+                   var metrics: Array[StructField]) {
+
```

Comment:
The same as DruidSchema dont use var use `case class`
```suggestion
class SparkSchema(
                   var dimensions: Array[StructField],
                   var metrics: Array[StructField]) {

```